app/src/test/java/ch/hikemate/app/model/route/saved/SavedHikesRepositoryDummyTest.kt

```diff
@@ -40,24 +40,36 @@ class SavedHikesRepositoryFirebaseTest {
   @Test
   fun addSavedHikeDoesNothing() =
```

Comment:
Really nitpicking: The name of the test is now incorrect 😅

Reply:
Oh right thank you for noticing I forgot about those

Reply:
It's actually important imo, test names should represent what the test does, so thanks haha